elevate-cpanel

```diff
@@ -1087,6 +1089,27 @@ sub run_stage_5 ($self) {
     return REBOOT_NEEDED;
 }
 
+sub check_and_fix_grub ($self) {
+
+    # for an autofixer
+    # return unless -e q[/var/cpanel/version/elevate];
+
+    my $grub_conf;
+    {
+        local $/;
+        open( my $fh, '<', DEFAULT_GRUB_FILE ) or return;
+        $grub_conf = <$fh>;
+    }
+
```

Comment:
maybe check if `eth0` is here or the output from the leapp log to confirm something like:
```
Detected interface rename eth0 -> ens18.
```

Reply:
even better check the output from `/proc/cmdline`